src/compiler/scala/tools/nsc/ast/parser/MarkupParsers.scala

```diff
@@ -268,14 +268,15 @@ trait MarkupParsers {
             acc.clear()
           }
           for (t <- ts)
-            t.attachments.get[handle.TextAttache] match {
+            (t.attachments.get[handle.TextAttache] match {
               case Some(ta) =>
                 if (acc.isEmpty) pos = ta.pos
                 acc append ta.text
               case _        =>
                 emit()
                 buf += t
-            }
+            }): Unit
```

Comment:
IIRC, when I last left this branch I was trying to extract a test case from the bootstrap failure here (before I disabled the LUB with `: Unit`). Can't find that in my git repo, so I'd say I didn't get very far. Would be interesting to see if whether your refined version would survive a rollback of this hunk.